video/out/gpu/ra.h

```diff
@@ -51,6 +51,9 @@ enum {
     RA_CAP_BUF_RW         = 1 << 6, // supports RA_VARTYPE_BUF_RW
     RA_CAP_NESTED_ARRAY   = 1 << 7, // supports nested arrays
     RA_CAP_GLOBAL_UNIFORM = 1 << 8, // supports using "naked" uniforms (not UBO)
+    RA_CAP_GATHER         = 1 << 9, // supports textureGather in GLSL
+    RA_CAP_FRAGCOORD      = 1 << 10, // supports reading from gl_FragCoord
+    RA_CAP_LIMITED_GLSL   = 1 << 11, // only simple GLSL shaders are supported
```

Comment:
That's really unspecific, and shouldn't be, because everyone has to know exactly what it implies if he's setting or using the flag.

Reply:
Yeah, agreed. I couldn't think of a good name. It forces ``have_mglsl = false`` in video.c, where mglsl is 'modern GLSL,' but the real limitation is the instruction limit with ps_2_0 shaders. Shaders that are arbitrarily 'complex' might blow out the instruction limit and this is hard to measure, because it relies on the optimizers in glslang/spirv-tools/spirv-cross/d3dcompiler.dll.

Reply:
Fixed by killing RA_CAP_LIMITED_GLSL, which is probably for the best. This only affects 9_2 and 9_1 hardware, which is probably a fraction of a percent of users running hardware more than 10 years old.